style.css

```diff
@@ -503,6 +521,17 @@ h1 {
     align-self: self-start;
   }
 
+  .name-group {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    gap: 15px;
+  }
+
+  #fullName {
+    display: none;
```

Comment:
- [ ] It will be better 😊 if you fix the responsive of the project on all screens as you see here when I switch from the mobile version to the desktop or the tablet version the responsive is bad, so it will be better if you fix that to let the users be able to see your portfolio from all sizes of the screen 💯😃

- I make a video 😃 for you to see instead of the image to see when I switch from the different sizes that the response is not good for the eye, so let's take a look below 😊⤵️

| Your design| 
| --- |
|     ![Recording 2023-05-12 at 18 35 52](https://github.com/AsumVictor/Porfolio/assets/83360636/6f44a051-e8ff-44c4-b4ef-1715be007020)  |


- Also these are images for that 😊⤵️

| Original design | Your design| 
| --- | --- |
|    ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/38cd99b2-3bb8-45df-baa4-f9b2a652cdd8)  |    ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/555b13ca-31e2-4b62-9885-2cb7282b1015)   |


| Original  design | Your design| 
| --- | --- |
|   ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/27d776bd-454e-409a-9608-985d5ce8f85e)  |  ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/6b80ea83-bf6e-40ab-bb93-e8fbd6b41fac)   |